config/samples/flows_v1alpha1_flowcollector_dev.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: "network-observability"
+  ipfix:
+    cacheActiveTimeout: 60s
+    cacheMaxFlows: 400
+    sampling: 100
+  goflowkube:
+    kind: Deployment
+    replicas: 1
+    port: 2055
+    image: "quay.io/netobserv/goflow2-kube:main"
+    imagePullPolicy: Always
+    logLevel: trace
+    printOutput: true
+    healthPort: 8080
+  loki:
+    batchWait: 1s
+    batchSize: 102400
+    minBackoff: 1s
+    maxBackoff: 300s
+    maxRetries: 10
+    timestampLabel: TimeFlowEnd
+    staticLabels:
+      app: netobserv-flowcollector
+    instanceSpec:
+      size: 1x.small
+      replicationFactor: 1
+      storage:
+        secret:
+          name: test
+      storageClassName: gp2
+  kafka:
+    instanceSpec:
+      kafka:
+        config:
+          offsets.topic.replication.factor: 1
+          transaction.state.log.replication.factor: 1
+          transaction.state.log.min.isr: 1
+          default.replication.factor: 1
+          min.insync.replicas: 1
+          inter.broker.protocol.version: "3.1"
+        storage:
+          type: ephemeral
+        listeners:
+          - name: plain
+            port: 9092
+            type: internal
+            tls: false
+          - name: tls
+            port: 9093
+            type: internal
+            tls: true
+        version: 3.1.0
+        replicas: 1
+      zookeeper:
+        storage:
+          type: ephemeral
+        replicas: 1
+  grafana:
+    instanceSpec:
+      config:
+        security:
+          admin_user: admin
+          admin_password: admin
+      ingress: true
+    dashboardSpec:
+      configMapRef:
+        name: grafana-dashboard
+        key: json
+        optional: true
+  prometheus:
+    instanceSpec:
+      replicas: 1
+  consolePlugin:
+    image: "quay.io/netobserv/network-observability-console-plugin:main"
+    imagePullPolicy: Always
+    port: 9001
+  cno:
+    namespace: "openshift-network-operator"
+  op: loki-operator,strimzi-kafka-operator,grafana-operator
```

Comment:
`op` isn't necessary anymore, right?
Also, `cno` was renamed `clusterNetworkOperator` (as a general rule, we should avoid cryptic names here, as this is something exposed to the end users)

Reply:
yes sorry it seems I missed that file when I merged :angel: